### PR TITLE
21-10210 Automated tests

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Add Unit- & E2E-tests to 21-10210 Lay/Witness Statement form-app.
- Team: VA Product Forms
- Not using Flipper

## Related issue

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team-forms#249

## Testing done

- Ran all tests successfully on local.

### CODE REVIEW

File: src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/feature-toggles.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "data": {
+    "type": "feature_toggles",
+    "features": []
+  }
+}
```
Comment (gmrabian) [inline src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/feature-toggles.json:6]: We have a shared feature-toggles we could use instead. It enables 26-4555 but should not affect this form. I figure we can enable every form that has a flipper in there as needed so tests run.

File: src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/form-submit.json
Comment (gmrabian) [inline src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/form-submit.json:1]: This file is also in the shared folder already

**Review by gmrabian — COMMENTED** [review]
A few suggestions for reusing shared files :)
Otherwise looks good! Love those reusable tests ;)

File: src/applications/simple-forms/21-10210/tests/e2e/10210-lay-witness-statement.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              selector: 'button',
+            }).click();
+          });
+        });
+      },
+    },
+
+    setupPerTest: () => {
+      // Log in if the form requires an authenticated session.
+      // cy.login();
```
Comment (gmrabian) [inline src/applications/simple-forms/21-10210/tests/e2e/10210-lay-witness-statement.cypress.spec.js:51]: remove if not needed

Comment (tlei123) [reply src/applications/simple-forms/21-10210/tests/e2e/10210-lay-witness-statement.cypress.spec.js:51]: Deleted.

File: src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/feature-toggles.json (hunk 1 above)
Comment (tlei123) [reply src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/feature-toggles.json:6]: Now importing shared file.

File: src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/form-submit.json
Comment (tlei123) [reply src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/form-submit.json:1]: Now importing shared file.

**Review by rhasselle-oddball — APPROVED** [review]
looks great!

**Pull request merged**